test/math.jl

```diff
@@ -1168,6 +1168,12 @@ end
             @test hypot(x, x*f, x*f) ≈ x * hypot(one(f), f, f) rtol=eps(T)
         end
     end
+    @testset "Real and Complex $T" for T in (Float16, Float32, Float64, BigFloat)
+        let x = sqrt(floatmax(T)/2 + 1)
```

Comment:
Thanks for the contribution!

Due to floating point rounding, `floatmax(T)/2 + 1 == floatmax(T)/2`, so this won't actually exercise the branch.

Should just test 
```suggestion
        let x = floatmax(T)
```